### PR TITLE
Droopler 3.3.0-rc1

### PR DESCRIPTION
**Pull request by grzegorz-pietrzak-droptica**

### CODE REVIEW

File: droopler.install [hunk 1]
```diff
@@ -2256,3 +2256,12 @@ function droopler_update_8138(&$sandbox) {
   \Drupal::service('d_update')->checklist()
     ->saveProgress(['v3.1' => $values]);
 }
+
+/**
+ * Droopler 3.3 Change AdvAgg JS minifier from JSqueeze to JShrink.
+ */
+function droopler_update_8139(&$sandbox) {
```
Comment (damian-sikora-droptica) [inline droopler.install:2263]: Maybe the update number should be 10001? But then maybe there can be an issue with overlapping number if somebody will update to Droopler 4?

File: modules/custom/d_p/d_p.libraries.yml [hunk 2]
```diff
@@ -6,16 +6,18 @@ d_p:
...
-  js:
-    js/d_p_ckeditor.js: {}
-    js/d_p_colorpicker.js: {}
-  dependencies:
-    - core/jquery
-    - core/jquery.once
-    - core/drupal
+#    - d_p/d_p_ckeditor
+
+# todo Adjust to Ckeditor5
```
Comment (grzegorz-pietrzak-droptica) [inline modules/custom/d_p/d_p.libraries.yml:11]: Can we remove it?

File: modules/custom/d_p/d_p.module [hunk 3]
```diff
@@ -137,8 +137,7 @@ function d_p_preprocess_paragraph(&$variables) {
  * Implements hook_form_alter().
  */
 function d_p_form_alter(&$form, FormStateInterface $form_state, $form_id) {
-  $form['#attached']['library'][] = 'd_p/d_p_ckeditor';
-
+  // $form['#attached']['library'][] = 'd_p/d_p_ckeditor';
```
Comment (grzegorz-pietrzak-droptica) [inline modules/custom/d_p/d_p.module:140]: We should remove it too, if it is not necessary

File: modules/custom/d_content/modules/d_content_init/src/ContentInitManagerBlock.php [hunk 4]
```diff
@@ -254,7 +254,7 @@ protected function placeBlockInWaMegaMenu(array $block, EntityInterface $block_e
         $submenu_config->rows_content[$row][$col] = $child_item;
         $menu_config->menu_config->{$parent_uuid} = $submenu_config;
         WeMegaMenuBuilder::saveConfig($menu_name, $theme, json_encode($menu_config));
-        we_megamenu_flush_render_cache();
+        \Drupal::cache('render')->invalidateAll();
```
Comment (grzegorz-pietrzak-droptica) [inline modules/custom/d_content/modules/d_content_init/src/ContentInitManagerBlock.php:257]: Please use DI here

**Review by damian-sikora-droptica — APPROVED** [review]

**Pull request merged**